module/behaviour/strategy/KeyboardWalk/src/KeyboardWalk.cpp

```diff
@@ -133,14 +136,25 @@ namespace behaviour {
             log("getup");
         }
 
-        void KeyboardWalk::kickRightForward() {
-            /*emit(std::make_unique<KickCommand>(KickCommand{
-                {-0.05, 0, 0}, //Ball is right of centre for right kick
-                {1, 0, 0}
-            }));*/
-            log("right forward kick");
+        void KeyboardWalk::kickLeft() {
```

Comment:
Is it worth making this a single function that takes in `LimbID` as a parameter?

Reply:
Done!